app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> int or str:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: float, average_rating: float,
+                 count_of_ratings: float) -> int or float:
```

Comment:
Fix type annotations, `clean_power` and `count_of_rating` aren't ` float`.
And the same as previous comment